web/client/translations/data.de-DE

```diff
@@ -691,6 +691,7 @@
             "export": "Export",
             "selectall": "Alle auswählen",
             "deselectall": "Auswahl aufheben",
+            "backtosearch": "Back to search",
```

Comment:
Maybe the right place to trigger this action is inside the toggleMenu actionCreator, using thunk to dispatch both actions.